static/img/diagrams/case-management-system-context.iuml

```diff
@@ -25,10 +26,14 @@ Rel(data_insight_team, qlik, "Manage data extraction and reporting pipelines")
 Rel(qlik, social_care_cases, "Hourly data export")
 Rel(google, qlik, "Data ingestion via Google API")
 Rel(social_care_cases, google, "Authenticates with groups and view Google Docs")
+Rel(social_care_workflows, google, "Authenticates with groups and view Google Docs")
 Rel(external_referrers, google, "Submit referrals via public form")
 BiRel(google, mash_referral, "Proxy referrals. Generate google docs")
 Rel(mash_referral, social_care_cases, "Submit to")
 BiRel(social_care_finance, social_care_cases, "TBD")
+Rel(care_practitioners, social_care_workflows, "Start an assessment")
+BiRel(social_care_cases, social_care_workflows, "Share resident data")
```

Comment:
is this the current production state? Should this diagram present production state only? I can see this is already out of date on other areas like MASH, so will probably require further tweaking to get it up to date.

Reply:
Yeah, there are changes to make to get it up to date, also things around finance are changing. I've scoped this PR to only add the core pathways stuff, nothing more.